src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -579,6 +579,13 @@ private async Task StopAsyncCore(bool disposing)
                         TaskContinuationOptions.ExecuteSynchronously | TaskContinuationOptions.OnlyOnFaulted,
                         TaskScheduler.Default);
                 }
+
+#pragma warning disable CA2252 // This API requires opting into preview features
```

Comment:
Does this mean that if we do make API changes to preview features in 9.0, it could make things break harder when mixing 8.0 and 9.0 packages?

Reply:
So like the following examples:
* If you used 8.0 client on the server that targets 9.0
* If you use Connection.Abstractions 9.0 with 8.0 client

Yeah that does seem bad :/
Generally we say "use the same version packages" since we ship everything at the same time. But it's sort of a cop-out.